doc/FEATURES.md

```diff
@@ -205,7 +207,17 @@ to production.
 
 ## DINTO Logfile Generation - Complete
 
-See [On Screen DINTO Reporting](#on-screen-dinto-reporting---complete)
+### Description
+A record of queries made to DINTO (via Chiron/Asclepius) can be found in the console logs of the Chiron service.
+
+### Testing
+This feature is tested manually.
+
+First, open up the tail of the panacea logs:
```

Comment:
Yes! Good idea. I also just noticed that i left "panacea" in from when I copy-pasted! Thanks!